output-management/web-viewer/downloader/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "devDependencies": {
-    "@openapitools/openapi-generator-cli": "2.8.0"
```

Comment:
Please use/[install](https://stackoverflow.com/questions/41003470/npm-install-the-exact-package-version-specified-in-package-json) fixed versions.